Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs

```diff
@@ -127,6 +127,9 @@ public enum ClearScreenOptions
     private static bool isButtonDoubleClicked;
 
     //private static MouseFlags? lastMouseButtonReleased;
+    // QUESTION: What's the difference between isButtonClicked and isButtonPressed?
```

Comment:
Clicked means "down then up"
Pressed means "down"

Reply:
> Clicked means "down then up" Pressed means "down"

Only if they have the same coordinates, otherwise will be `Pressed` and `Released`.

Reply:
If `isButtonPressed` is true and a `Released` event occur on the same coordinates the a `Clicked` event will fire and set the 
`isButtonClicked` to true. Then if another Pressed and Release occur on the same coordinates in a lower space of milliseconds a double clicked will occur and `isButtonDoubleClicked` is true. Then if another Pressed and Release occur on the same coordinates in a lower space of milliseconds a triple clicked will occur.

Reply:
Thanks. :)